ui/src/dashboards/components/DashboardsPageContents.tsx

```diff
@@ -27,12 +22,12 @@ interface Props {
   onExportDashboard: (dashboard: Dashboard) => () => void
   onImportDashboard: (dashboard: Dashboard) => void
   notify: (message: Notification) => void
-  showOverlay: ShowOverlayActionCreator
   dashboardLink: string
 }
 
 interface State {
   searchTerm: string
+  showOverlay: boolean
```

Comment:
how would you feel about changing boolean variable names to be prefixed with `is` such as `isOverlayShown` to make it readable as obviously a boolean?